projects/1.19/assets/quark/quark/lang/zh_cn.json

```
@@ -1,1296 +1,1554 @@
+	"quark.jei.hint.seed_pouch": "可以容纳一堆任意同种种子，只需右击即可存入/取出。主手或副手拿着种子袋时，它会自动收起你拾取的同种种子，潜行状态下对耕地使用可播种3x3的区域。（树苗等也算种子）",
+	"quark.jei.hint.shale": "可以在积雪的生物群系中找到它的大型团簇。",
+	"quark.jei.hint.slime_in_a_bucket": "对小型史莱姆使用桶，将其装入桶内。当你在史莱姆区块中时，它会开始跳动。",
+	"quark.jei.hint.soul_bead": "由灵魂沙峡谷的游魂掉落。使用时会飞向最近的下界要塞。",
+	"quark.jei.hint.sturdy_stone": "不能用活塞移动。",
+	"quark.jei.hint.tiny_potato": "它信任你！",
+	"quark.jei.hint.torch_arrow": "无论它射中何处，都会尝试在那里放置火把。",
+	"quark.jei.hint.trowel": "使用时随机放置快捷栏中的方块。让一种方块占据多个格子，可以增加该种方块的放置几率。",
+	"quark.jei.hint.yellow_blossom_sapling": "可以在平原系生物群系中找到。",
+	"quark.jei.hint.ancient_tome": "可以将物品/附魔书的魔咒提升一级，最多超过正常上限一级。偶尔能在地牢或要塞中找到它。",
+	"quark.jei.hint.hollow_logs": "走向空洞一侧，进入潜行状态就会钻进去，而且里面可以爬。",
+	"quark.jei.hint.skull_pikes": "放在栅栏上面时，能吓跑附近5格范围内徘徊的简单怪物。（简单怪物指除Boss/袭击者以外的怪物）",
+	"quark.jei.hint.pottable_stuff": "现在可以放进花盆里了。",
+	"quark.jei.hint.stools": "可以坐在上面，并作为整体被活塞移动。在上面放置方块后，凳子会变成桌子。",
+	"quark.jei.hint.posts": "可以朝三个坐标轴方向放置。锁链和灯笼可以连接到它们。",
+	"quark.jei.hint.soul_fire_candles": "有些方块点燃后会产生灵魂火，将它放在这些方块上，点燃后同样会产生灵魂火。",
+	"quark.jei.hint.wool_muffling": "在羊毛上放置铁轨，可以使矿车静音。",
+	"quark.jei.hint.shulker_box_right_click": "类似收纳袋，可以右击将物品收入潜影盒。",
+	"quark.jei.hint.rotating_bundles": "在收纳袋上悬停时，滚动鼠标滚轮，可循环内部槽位。",
+	"quark.jei.hint.crab_info": "螃蟹可以在沙滩系生物群系中找到。可以用小麦、鸡肉或任何鱼来繁殖它们",
+	"quark.jei.hint.ambience_discs": "在骷髅杀死蜘蛛时可以获得环境唱片。与音乐唱片不同，它们会无限循环播放。",
+	"quark.jei.hint.runes": "在铁砧中，将它们与附魔物品结合，可以改变物品的光效颜色。这是纯视觉效果。",
+	"quark.jei.hint.parrot_eggs": "丢出后，会造出一只幼年鹦鹉。喂食大量甜菜根种子来驯服鹦鹉，然后它下一颗蛋需要10分钟。",
+	"quark.jei.hint.doors_open_together": "制造一对双开门后，它们可以一起开启/关闭。",
+	"quark.jei.hint.fence_gates_open_together": "把两个栅栏门上下堆叠放置，它们会一起打开/关闭",
+	"quark.jei.hint.item_frame_dyeing": "可以染色，染色方式和皮革盔甲相同。",
+	"quark.jei.hint.gold_tool_fortune": "金工具会内置（%s级）时运或抢夺的效果。这效果不会与魔咒的叠加，高于内置效果等级的魔咒才能覆盖它。",
+	"quark.jei.hint.gold_tool_harvest_level": "现在金工具的挖掘等级为%s。",
+	"quark.jei.hint.hoe_harvesting": "锄可以一下子破坏3x3的区域内的植被。较高品质的锄可以破坏5x5的区域。",
+	"quark.jei.hint.head_sfx": "任何怪物的头颅都可以放置到音符盒的侧面，之后音符盒在激活时会发出它们的音效。",
+	"quark.jei.hint.sign_editing": "告示牌可以右击编辑。",
+	"quark.jei.hint.chorus_weeds": "在末地荒地偶尔会生成，末地荒地位于末地外岛。会随时随机传送和增殖，或者通过踩踏、骨粉或附近的水流强迫它这么做。此外，踩踏时可能生成末影螨。",
+	"quark.jei.hint.corundum": "可以在地下发现一大簇，每一大簇都有两种颜色。如果放在地下深处，会慢慢向上生长。可以涂蜡来阻止生长和粒子产生。",
+	"quark.jei.hint.crystal_lamp": "这些灯只有在激活时才会改变信标光束的颜色。",
+	"quark.jei.hint.snow_golem_player_heads": "让女巫杀死被命名的雪傀儡时，若雪傀儡戴着雕刻过的南瓜，会掉落一个同名玩家的头。",
+	"quark.jei.hint.use_for_vanishing": "雕刻过的南瓜若附有消失诅咒，在佩戴时不会显示原本的覆盖层。",
+	"quark.jei.hint.use_for_binding": "玩家的头若附有绑定诅咒，放在盔甲架上会显示对应玩家的全身模型。",
+	"quark.jei.hint.pipe": "会移动物品，物品可以用漏斗/投掷器或类似物放入，或是通过将物品丢进管道开口放入。物品移动时会优先下落，其次向前，然后向随机侧面，最后才是向上。将进入已满容器的物品会返回。可以通过红石信号禁用任何管道，还可以使用红石比较器读取管道中的物品数量。内部超过16个物品的管道会破裂。",
+	"quark.jei.hint.magnet": "充能后，可以推/拉任何制作时用到铜/铁的方块，具体推还是拉取决于是哪一面。如果多个磁铁在移动同一个方块，则力最大的一方生效。磁铁施加于方块的力=它接收到的红石信号等级-它到方块的距离。漏斗用磁铁移动时，每移动一格，都会在前方掉落一件内容物。如果该物品是种子，它会尝试种植该种子。切石机用磁铁移动时，会尝试破坏它们上方的方块。",
```

```suggestion
	"quark.jei.hint.magnet": "充能后，可以推/拉任何制作时用到铜/铁的方块，具体行为取决于侧面磁极。如果多个磁铁在移动同一个方块，则力最大的一方生效。磁铁施加于方块的力的大小等于它接收到的红石信号强度减去它到方块的距离。漏斗用磁铁移动时，每移动一格，都会在前方掉落一件内容物。如果该物品是种子，它会尝试种植该种子。切石机被磁铁移动时，会尝试破坏它们上方的方块。",
```
或者这么写：
磁铁施加于方块的力的大小 = 红石信号强度 - 到方块的距离
数学格式以空格分割